File: 2-SortingTheSentence/sortingTheSentence.py
class Solution:
    def sortSentence(self, s: str) -> str:
        spli = s.split(' ')
        res = ''
        counter = 0
        for y in spli:
            counter += 1
            for s in spli:
                if (s[-1] == str(counter)):
                    res += s.replace(str(counter), ' ')
                    break
        return res.strip()

File: 2-SortingTheSentence/sortingTheSentence.js
/**
 * @param {string} s
 * @return {string}
 */
var sortSentence = function(s) {
    s = s.split(' ')
    let result = ''
    let counter = 0
    for (const word of s) {
        counter += 1
        for (const w of s) {
            if (parseInt(w.split('').reverse()[0]) === counter) {
                result += w.replace(counter.toString(), ' ')
            }
        }
    }
    return result.trim()
};
